hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -80,6 +80,7 @@ class RobotHW : public InterfaceManager
    */
   virtual bool init(ros::NodeHandle& /*root_nh*/, ros::NodeHandle &/*robot_hw_nh*/) {return true;}
 
+
```

Comment:
It doesn't look like the rest of the codebase is entirely consistent, but I think this extra newline should be removed to be in line with the majority.